tutorials/introductory/images.py

```diff
@@ -242,19 +235,17 @@
 # We'll use the Pillow library that we used to load the image also to resize
 # the image.
 
-from PIL import Image
-
 img = Image.open('../../doc/_static/stinkbug.png')
 img.thumbnail((64, 64), Image.ANTIALIAS)  # resizes image in-place
```

Comment:
```suggestion
img.thumbnail((64, 64))  # resizes image in-place
```
May just as well put that here instead of #23881 (which can be closed then).

Reply:
I merged yours.